<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: radial-gradient( circle 364px at -6% -6.4%,  rgba(95,75,139,1) 16.9%, rgba(128,204,221,1) 88.5% );</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_song_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;mmmm&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: rgb(255, 255, 255);
font: 15pt &quot;Segoe UI&quot;;
font: 700 13pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>Playing Now</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pause_play">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_prev">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_seek_bar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>571</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color::rgb(49, 41, 61)</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_volume">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_timer">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>341</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(74, 81, 81)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 30pt &quot;ORGANICAL PERSONAL USE&quot;;</string>
    </property>
    <property name="text">
     <string>FAITH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pablobutton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sherenbutton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="amrbutton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="zazabutton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fayrouzbutton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="weekndbutton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 50pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuopen">
    <property name="title">
     <string>open</string>
    </property>
    <addaction name="actionopen_audio_file"/>
   </widget>
   <widget class="QMenu" name="menuplaylist">
    <property name="title">
     <string>playlists</string>
    </property>
    <addaction name="actionname_playlist"/>
   </widget>
   <widget class="QMenu" name="menunew_playlist">
    <property name="title">
     <string>new playlist</string>
    </property>
   </widget>
   <addaction name="menuopen"/>
   <addaction name="menuplaylist"/>
   <addaction name="menunew_playlist"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionname_playlist">
   <property name="text">
    <string>name playlist</string>
   </property>
  </action>
  <action name="actionadd_new_playlist">
   <property name="text">
    <string>add new playlist</string>
   </property>
  </action>
  <action name="actionopen_audio_file">
   <property name="text">
    <string>open audio file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
